android/build.gradle

```diff
@@ -40,6 +40,7 @@ android {
         implementation "com.mapbox.mapboxsdk:mapbox-android-sdk:9.2.0"
         implementation "com.mapbox.mapboxsdk:mapbox-android-plugin-annotation-v9:0.8.0"
         implementation "com.mapbox.mapboxsdk:mapbox-android-plugin-localization-v9:0.12.0"
+        implementation "com.mapbox.mapboxsdk:mapbox-android-plugin-offline-v8:0.6.0"
```

Comment:
Could you use offline-v9 instead? Plugins have the major version of the maps SDK appended to them.